### DISCUSSION TITLE
Can't install from eclipse marketplace

### DISCUSSION THREAD

**Discussion by ccartee** [subclipse/subclipse#273] [Q&A] [answered]
link goes to 404 https://subclipse.github.io/updates/subclipse/4.3.x/

health has same link
https://www.eclipse.org/setups/marketplace/?id=979

**Comment by markphip** [comment]
Working fine for me: https://subclipse.github.io/updates/compositeContent.xml

It is just a GitHub Pages site so not much we can do about it being up or down

**Comment by ccartee** [comment]
Unable to read repository at https://subclipse.github.io/updates/compositeContent.xml/content.xml.
Unable to read repository at https://subclipse.github.io/updates/compositeContent.xml/content.xml.
PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target

**Comment by ccartee** [comment, marked as answer]
Tried another machine.  It worked there. I suspect this is some sort of conflcit with zscaler

Addressed by adding zscalar proxy as trustyed ca cert for java

https://help.zscaler.com/zia/adding-custom-certificate-application-specific-trust-store

from java root in eclipse:
bin\keytool  -import  -trustcacerts -alias zscalerRoot -file zscaler.der.cer -keystore lib\security\cacerts
